pylint/checkers/refactoring.py

```diff
@@ -654,9 +654,12 @@ def visit_call(self, node):
             # we're finally out of any nested boolean operations so check if
             # this len() call is part of a test condition
             if _node_is_test_condition(parent):
+                try:
+                    node is parent.test or parent.test.locate_child(node)
```

Comment:
my intention is to check whether node belongs to `test` node of conditional statement.
Is there a better way for doing it?